tests/pom/editor/Editor.ts

```diff
@@ -34,6 +40,17 @@ export class Editor {
     await expect(this.topToolbar).toContainText(this.secondaryPageName);
   }
 
+  getElement(content?: string) {
+    return content ?
+      this.page.locator('.content-element', { hasText: content }) :
```

Comment:
Since this selector is being repeated across multiple files consider extracting it as static property of ContentElement pom ;)